### PR TITLE
TELCODOCS:1103b - Update to kataconfig file in 4.11

### PR DESCRIPTION
**Pull request by MirzWeiss**
Version(s): 4.11

Issue: [TELCODOCS-1103](https://issues.redhat.com/browse/TELCODOCS-1103)

[Link to docs preview](https://55840--docspreview.netlify.app/openshift-enterprise/latest/sandboxed_containers/deploying-sandboxed-container-workloads.html#sandboxed-containers-create-kataconfig-resource-web-console_deploying-sandboxed-containers)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by tbuskey** [comment]
QE ack /lgtm

**Comment by eohartman** [comment]
Looks great!

File: modules/sandboxed-containers-create-kataconfig-resource-web-console.adoc [hunk 1]
```diff
@@ -66,8 +66,9 @@ spec:
   kataMonitorImage: registry.redhat.io/openshift-sandboxed-containers/osc-monitor-rhel8:{sandboxed-containers-version}.0
   kataConfigPoolSelector:
     matchLabels:
-      <label_key>: '<label_value>'
+      <label_key>: '<label_value>' <1>
 ----
+<1> Labels in `kataConfigPoolSelector` only support single values; `nodeSelector` syntax is not supported.
```
Comment (kelbrown20) [inline modules/sandboxed-containers-create-kataconfig-resource-web-console.adoc:71]: Do you think its worth adding just like a example of a label to use? 

```suggestion
<1> Labels in `kataConfigPoolSelector` only support single values, including <example>. The `nodeSelector` syntax is not supported.
```

**Review by kelbrown20 — COMMENTED** [review]
Merging!

**Pull request merged**